Documentation/design/developing.md

```diff
@@ -1,8 +1,8 @@
  # Setting up a namespaced OLM for development
  
  * Disable global olm/catalog if the cluster is only for testing
-    * spin `tectonic-alm-operator` down to 0
-    * delete `alm-operator` and `catalog-operator` deployments from `tectonic-system`
+    * spin `tectonic-OLM-operator` down to 0
```

Comment:
RIP Tectonic :cry:  We should change this section to be more generic for upstream Kubernetes/OpenShift.

Reply:
Totally agree - but I think we should address doc cleanup in a future PR. I was hoping to keep this one tightly scoped to `OperatorGroup`s. I'm pulling these changes out for now.